src/styles/components/_usage-guidelines.scss

```diff
@@ -52,40 +52,71 @@ $breakpoint-usage-guidelines-display: 70rem;
     }
   }
 
-  p {
-    font-size: map.get($font-sizes, "xs");
-    line-height: map.get($line-heights, "xl"); // 28px/16
+  &__content {
+    p {
+      line-height: 1.875rem;
+      font-size: map.get($font-sizes, "xs");
 
-    @media (min-width: $breakpoint-usage-guidelines-display) {
-      font-size: map.get($font-sizes, "sm");
-      line-height: map.get($line-heights, "lg"); // 28px/20
+      @media (min-width: $breakpoint-usage-guidelines-display) {
+        font-size: map.get($font-sizes, "sm");
+      }
     }
-  }
 
-  ul {
-    margin-left: 1.5rem;
-  }
+    li {
+      position: relative;
+      font-size: map.get($font-sizes, "xs");
+      line-height: 1.75; // 28px/16
 
-  li {
-    position: relative;
-    font-size: map.get($font-sizes, "xs");
-    line-height: map.get($line-heights, "xl"); // approx. 28px/16
-
-    &::before {
-      content: "";
-      background-image: url("../../../public/images/list_markers/emdash.svg");
-      background-repeat: no-repeat;
-      background-size: 80%;
-      position: absolute;
-      width: 1.25rem;
-      height: 1rem;
-      left: -1.75rem;
-      top: 0.8rem;
+      &::before {
+        content: "";
+        background-image: url("../../../public/images/list_markers/emdash.svg");
+        background-repeat: no-repeat;
+        background-size: 80%;
+        position: absolute;
+        width: 1.25rem;
+        height: 1rem;
+        left: -1.75rem;
+        top: 0.8rem;
+      }
+
+      @media (min-width: $breakpoint-usage-guidelines-display) {
+        font-size: map.get($font-sizes, "sm");
+        line-height: 1.5; // 30px/20
+      }
     }
 
-    @media (min-width: $breakpoint-usage-guidelines-display) {
-      font-size: map.get($font-sizes, "sm");
-      line-height: map.get($line-heights, "lg"); // approx. 28px/20
+    // The following is for targeting the outermost bullet points of the ul
+    // and giving them sub-header styles.
+    div > ul {
+      ::before {
+        display: none;
+      }
+
+      & > li {
```

Comment:
I was inconsistent with the '&' syntax here. Not sure if with or without the ampersand is more appropriate, but I should pick one and stick with it.